src/file_io.py

```diff
@@ -10,9 +10,18 @@
 
 # YOUR CODE HERE
 
+with open('foo.txt', 'r') as f:
```

Comment:
Nice job! one more easy way was to just create a variable with the open() function and then print it out with the read() function.